packages/sfui/frameworks/react/components/SfSelect/SfSelect.tsx

```diff
@@ -66,6 +66,7 @@ export default function SfSelect(props: SfSelectProps) {
           <option
             disabled
             hidden
+            selected
```

Comment:
This is breaking change as when user will add their option in slot with selected option, then user selected option wont be marked as selected because hidden placeholder is selected

Reply:
@Szymon-dziewonski thank you for finding the best possible solution. Turned out using the `disabled` and `hidden` attrs simultaneously was causing this issue. I've corrected that [here](https://github.com/vuestorefront/storefront-ui/pull/3263/commits/a8611fdded63dc087287ae4986f67aae24f4b14c).